schema/messaging-experiments.schema.json

```diff
@@ -52,7 +52,6 @@
               },
               "value": {
                 "description": "Message content",
-                "type": "object",
```

Comment:
Let's add the "null" type as one of the valid values here.

```json
 "value": {
    "description": "Message content",
    "oneOf": [
      {
         "type": "object",
         "properties": {}
      },
      {
        "type":  "null"
      }
  ]
},
```

Reply:
Updated patch with feedback, Thanks!